sdk/communication/azure-communication-chat/tests.yml

```diff
@@ -5,3 +5,18 @@ extends:
   parameters:
     PackageName: azure-communication-chat
     SafeName: azurecommunicationchat
+    CloudConfig:
+      Public:
+        SubscriptionConfigurations:
+          - $(sub-config-azure-cloud-test-resources)
+          - $(sub-config-communication-services-cloud-test-resources-common)
+          - $(sub-config-communication-services-cloud-test-resources-java)
+        SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
+        ServiceConnection: azure-sdk-tests
+      Int:
+        SubscriptionConfigurations:
+          - $(sub-config-communication-int-test-resources-common)
+          - $(sub-config-communication-int-test-resources-java)
+        ServiceConnection: azure-sdk-tests
```

Comment:
The `ServiceConnection` for INT env is pending on EngSys.

Reply:
Note: Currently "azure-communication-chat" lib in _main_ runs in Public and INT, so those two CloudConfigs are added in this file. (There are other azure-communication-* (e.g., email) libs that also runs in PPE in addition to Public and INT)